src/recensio/plone/browser/review.py

```diff
@@ -530,6 +532,42 @@ def getLicense(self):
                 current = current.aq_parent
         return True and publication_licence or _("license-note-review")
 
+    def getUUIDUrl(self):
+        base_url = api.portal.get().absolute_url()
+        if base_url.startswith("http://www."):
+            base_url = base_url.replace("http://www.", "http://")
+        base_url += "/r/"
+        base_url += self.context.UID()
+        return f'<a href="{base_url}">{base_url}</a>'
+
+    def get_citation_location(self):
+        location = []
+        try:
+            doi_active = self.isDoiRegistrationActive()
```

Comment:
This will try/except mask the fact that the view has no isDoiRegistrationActive method defined